support/camel-k-maven-plugin/src/main/java/org/apache/camel/k/tooling/maven/GenerateCatalogMojo.java

```diff
@@ -464,7 +465,11 @@ private void addCapabilities(RuntimeSpec.Builder runtimeSpec, CamelCatalogSpec.B
 
         artifacts.clear();
         artifacts.add(Artifact.from("org.apache.camel.quarkus", "camel-quarkus-microprofile-health"));
-        addCapabilityAndDependecies(runtimeSpec, catalogSpec, "health", artifacts, false);
+        List<Property> properties = new ArrayList<>();
+        properties.add(Property.from("defaultLivenessProbePath", "/q/health/live"));
+        properties.add(Property.from("defaultReadinessProbePath", "/q/health/ready"));
+        properties.add(Property.from("defaultStartupProbePath", "/q/health/started"));
```

Comment:
nitpick: this does not seems a real property but more an endpoint

Reply:
True. We can use a generic metadata bucket of properties if you think it's worth. Let me rework this part, thanks.

Reply:
the main issue is about cognitive load as one has to take into account that for the health, they have a different meaning than for other traits

Reply:
No, I think it would be an additional configuration, not a replacement. The same `metadata` can be provided for other stuff that may be required with the meaning of "generic" configuration that have to be managed by the operator.

Reply:
I meant, if we leave them as `properties`

Reply:
Ah, yes, I was agreeing with you. I do sometime :)